fabtests/scripts/runfabtests.py

```diff
@@ -201,6 +201,11 @@ def fabtests_args_to_pytest_args(fabtests_args):
         pytest_args.append("-k")
         pytest_args.append(fabtests_args.expression)
 
+    if fabtests_args.html:
+        pytest_args.append("--html")
```

Comment:
Are these new args? Has the behavior not been implemented yet?

Reply:
Not sure what is the question?

Reply:
Ah okay, didn't realize it was a native pytest arg since it was added to fabtests_args